pkg/apiserver/statement/service.go

```diff
@@ -195,7 +200,7 @@ func (s *Service) plansHandler(c *gin.Context) {
 		return
 	}
 	db := utils.GetTiDBConnection(c)
-	plans, err := QueryPlans(db, req.BeginTime, req.EndTime, req.SchemaName, req.Digest)
+	plans, err := QueryPlans(db, s.params.SysSchema, req.BeginTime, req.EndTime, req.SchemaName, req.Digest)
```

Comment:
maybe you should make it `s.QueryPlans(db, req.BeginTime, req.EndTime, ...)`, which could be cleaner :)

Previously there is no state to be hold by QueryPlans, so that it is natural to be a standalone function. Now it needs some service lifecycle variables, and then it deserves to be a service function instead.